connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/RelationalGroupedDataset.scala

```diff
@@ -109,7 +109,7 @@ class RelationalGroupedDataset protected[sql] (
     agg(exprs.asScala.toMap)
   }
 
-  private[this] def strToExpr(expr: String, inputExpr: proto.Expression): proto.Expression = {
+  private[this] def strToColumn(expr: String, inputExpr: proto.Expression): Column = {
```

Comment:
How about:
```scala
private[this] def strToColumn(expr: String, inputExpr: Column): Column = {
  expr.toLowerCase(Locale.ROOT) match {
    case "avg" | "average" | "mean" => functions.avg(inputExpr)
    case "stddev" | "std" => functions.avg(inputExpr)
    case "count" | "size" => functions.count(inputExpr) // Analyzer will take care of * expansion
    case name => Column.fn(name, inputExpr)
  }
}
```

Reply:
I see what you are suggesting now. Done.